tests/CoaddPsf.py

```diff
@@ -534,6 +510,48 @@ def testValidPolygonPsf(self):
             self.assertTrue(testRelDiff(m1, m1coadd, 0.01))
             self.assertTrue(testRelDiff(m2, m2coadd, 0.01))
 
+    def testGoodPix(self):
+        """Demonstrate that we can goodPix information in the CoaddPsf"""
+        cd11, cd12, cd21, cd22  = 5.55555555e-05, 0.0, 0.0, 5.55555555e-05
+        crval1, crval2 = 0.0, 0.0
+        bboxSize = afwGeom.Extent2I(2000, 2000)
+        crpix = afwGeom.PointD(bboxSize/2.0)
+        crval = afwCoord.Coord(afwGeom.Point2D(crval1, crval2))
+        wcsref = afwImage.makeWcs(crval, crpix, cd11, cd12, cd21, cd22)
+
+        schema = afwTable.ExposureTable.makeMinimalSchema()
+        schema.addField("weight", type="D", doc="Coadd weight")
+        schema.addField("goodpix", type="I", doc="Number of good pixels")
+        mycatalog = afwTable.ExposureCatalog(schema)
```

Comment:
Being especially picky, I might question whether some of the variable names here match the coding standard. Should it be `myCatalog`, `wSum`, etc?